Developed in-house, not fed back.
Solaris specific: manpage update to reflect Solaris specific
configuration.

--- doc/lighttpd.8-orig	Mon Mar 30 15:16:59 2009
+++ doc/lighttpd.8	Mon Jun 20 08:03:16 2011
@@ -47,10 +47,10 @@
 .
 .SH FILES
 .TP 8
-/etc/lighttpd/lighttpd.conf
+/etc/lighttpd/1.4/lighttpd.conf
 The standard location for the configuration file.
 .TP 8
-/var/run/lighttpd.pid
+/var/lighttpd/1.4/logs/run.pid
 The standard location for the PID of the running \fBlighttpd\fP process.
 .
 .SH SEE ALSO
